governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx

```diff
@@ -0,0 +1,411 @@
+                          ) : null}
+                          <div
+                            className="-ml-1 inline-flex cursor-pointer items-center px-1 hover:bg-dark hover:text-white active:bg-darkGray3"
+                            onClick={() => {
+                              copy(
+                                instruction.accounts.named[
+                                  key
+                                ].pubkey.toBase58()
+                              )
+                            }}
+                          >
+                            <span className="mr-2 hidden xl:block">
+                              {instruction.accounts.named[
+                                key
+                              ].pubkey.toBase58()}
+                            </span>
+                            <span className="mr-2 xl:hidden">
+                              {instruction.accounts.named[key].pubkey
+                                .toBase58()
+                                .slice(0, 6) +
+                                '...' +
+                                instruction.accounts.named[key].pubkey
+                                  .toBase58()
+                                  .slice(-6)}
+                            </span>{' '}
+                            <CopyIcon className="shrink-0" />
+                          </div>
+                        </div>
+                      </div>
+                    </>
+                  ))}
+                </div>
+              ) : (
+                <div>No arguments</div>
+              )}
+            </div>
+            {index !== proposalInstructions.length - 1 ? (
+              <hr className="border-gray-700" />
+            ) : null}
+          </>
+        ))}
+      </div>
+    </div>
+  ) : (
+    <div className="mt-3">
+      <Loadbar theme="light" />
+    </div>
+  )
+}
+
+const Proposals = () => {
+  const router = useRouter()
+  const [multisig, setMultisig] = useState<MultisigAccount>()
+  const [proposals, setProposals] = useState<TransactionAccount[]>([])
+  const [currentProposal, setCurrentProposal] = useState<TransactionAccount>()
+  const [currentProposalPubkey, setCurrentProposalPubkey] = useState<string>()
+  const [dataIsLoading, setDataIsLoading] = useState<boolean>(false)
+  const { cluster } = useContext(ClusterContext)
+  const anchorWallet = useAnchorWallet()
+  const { isLoading: isMultisigLoading, squads } = useMultisig(
+    anchorWallet as Wallet
+  )
+
+  const handleClickBackToPriceFeeds = () => {
+    delete router.query.proposal
+    router.push(
+      {
+        pathname: router.pathname,
+        query: router.query,
+      },
+      undefined,
+      { scroll: false }
+    )
+  }
+
+  useEffect(() => {
+    const fetchProposals = async () => {
+      setDataIsLoading(true)
+      if (squads) {
+        // TODO: support get proposals from other cluster when the security multisig is deployed
+        if (cluster === 'devnet') {
```

Comment:
Right now when I switch from devnet to mainnet, devnet proposals stay

Reply:
But I get that will be fixed when we deploy security multisig everywhere

Reply:
good catch, will fix